taichi/jit/jit_session.h

```diff
@@ -11,13 +11,15 @@ TLANG_NAMESPACE_BEGIN
 
 // Backend JIT compiler for all archs
 
+class LlvmProgramImpl;
+
 class JITSession {
  protected:
   std::vector<std::unique_ptr<JITModule>> modules;
+  LlvmProgramImpl *llvm_prog;
```

Comment:
Our [C++ style](https://docs.taichi.graphics/lang/articles/contribution/cpp_style) generally follows Google C++ Style Guide. According to https://google.github.io/styleguide/cppguide.html#Access_Control, it may be better to keep data members private and use protected accessors.